docker-compose.yml

```diff
@@ -45,17 +61,23 @@ services:
     build:
       context: .
       dockerfile: ./docker/Dockerfile-workers
-    # Override the default command so that Celery auto-reloads on code changes
+    # Override the default command so that Celery auto-reloads on code changes.
+    # This also adds the self-signed CA that was used to sign the Docker registry's certificate
+    # to the trusted CA bundle. This will make podman trust the Docker registry's certificate.
```

Comment:
how about "podman trust the Docker registry's certificate." => "podman trust the local Docker registry's certificate."